_products/02-formsg.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /products/formsg/
+title: Form.gov.sg
+image: /images/form-mockup.png
+breadcrumb: Form.gov.sg
+
+---
+
+
+![github](/images/form-mockup.png)
+
+### Build government forms in minutes
+
+#### Background 
+
+From applying for permits to booking public facilities, paper forms have been a vital tool for the public service to gather information. However, paper forms are logistically cumbersome, and public digital form builders like Google Forms present security risks as sensitive citizen information will be stored on the cloud.  
+
+#### Approach
+
+* Intuitive user interface allows new forms to be created in less than 30 minutes
+* Zero knowledge architecture allows for mass digitization without large security vulnerabilities
+* Pre-built integrations with Singpass, CorpPass and MyInfo extends functionality of forms
+
+
+#### Impact
+
+* 40k digital forms created
+* 50 million form submissions from 40k public officers across 90 agencies
+* $5k - $150k saved per form
+
+#### Team
+
+Antariksh Mahajan <a href="https://github.com/mantariksh" style="display: inline-block; width: 24px; height: 24px; margin-bottom: -5px; margin-left: 10px;">
+    <img border="0" alt="Github account" src="/images/Github-Mark-32px.png">
+</a>
+
+Arshad Samad <a href="https://github.com/arshadali172" style="display: inline-block; width: 24px; height: 24px; margin-bottom: -5px; margin-left: 10px;">
+    <img border="0" alt="Github account" src="/images/Github-Mark-32px.png">
+</a>
+
+Kar Rui Lau <a href="https://github.com/karrui" style="display: inline-block; width: 24px; height: 24px; margin-bottom: -5px; margin-left: 10px;">
+    <img border="0" alt="Github account" src="/images/Github-Mark-32px.png">
+</a>
+
+Leonard Loo <a href="https://github.com/leonardloo" style="display: inline-block; width: 24px; height: 24px; margin-bottom: -5px; margin-left: 10px;">
+    <img border="0" alt="Github account" src="/images/Github-Mark-32px.png">
+</a>
+
+Pearly Ong <a href="https://github.com/pearlyong" style="display: inline-block; width: 24px; height: 24px; margin-bottom: -5px; margin-left: 10px;">
```

Comment:
Would Github accounts only make sense for engineers? Dont know if Designers have Figma accounts. But linking to Github for Designers/PMs seems a bit misleading

Reply:
oh hmm let me ask the designers/pm's what they prefer